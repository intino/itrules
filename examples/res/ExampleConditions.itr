def type(Person)
    $Name has $PetsCount+Words pets:
      $Pets...[$NL]
end

def type(Dog) AND attribute(Name)
    $Name, a $Age dog
end

def type(Dog)
    A $Age dog
end

def type(Cat)
    $Name, a $Age+Age kitty
end

def trigger(Age) AND attribute(this, 1)
    one year old
end

def trigger(Age)
    $this+Words years old
end