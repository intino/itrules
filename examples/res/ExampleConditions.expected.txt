Pau Gasol has three pets:
  Ruffo, a five years old dog
  Missy, a one year old kitty
  A three years old dog